backend/utils/sidebar.py

```diff
@@ -0,0 +1,183 @@
+from utils.room_utils import DB, DEFAULT_DM_IMG, get_org_rooms
+
+
+class Sidebar:
+    """Serves as a data source for the sidebar
+
+    Sidebar class helps makes faster connection to core,
+    sorts out data, and creates the sidebar data format
+    for the frontend
+    """
+
+    @classmethod
+    async def __get_room_members(
+        cls, member_id: str, room: dict, org_members: list
+    ) -> dict:
+        """gets the room members excluding the current user
+
+        Args:
+            member_id (str): member_id of the current user
+            room (dict): room object data
+            org_members (list): list of all members in the organization
+
+        Returns:
+            [dict]: key value pair of room members
+                    example: {'member_id': {
+                                'username': str
+                                'image_url': str
+                                'starred': bool
+                                'role': str}
+                            }
+        """
+        room_members = room["room_members"]
+        room_members.pop(member_id)  # remove self from room members
+        room_members_ids = room_members.keys()
+        for room_member_id in room_members_ids:
```

Comment:
Here, what if we do directly like this:
```suggestion
        for room_member_id in room_members.keys():
```
And then we can remove line 34

Reply:
Fixed